src/number_theory/arithmetic_function.lean

```diff
@@ -530,6 +534,89 @@ begin
   apply ((is_multiplicative_zeta).mul is_multiplicative_pow)
 end
 
+/-- `Ω n` is the number of prime factors of `n`. -/
+def card_factors : arithmetic_function ℕ :=
+⟨λ n, n.factors.length, rfl⟩
+
+localized "notation `Ω` := card_factors" in arithmetic_function
+
+@[simp]
```

Comment:
Are you sure you want this to be `simp`? The right hand side doesn't look very simple to me, compared to the LHS.
I suggest that people add this to the simp set manually, if they want to use it.

Reply:
I agree, this doesn't seem like a natural simp lemma.

Reply:
I've changed these to the definitional equality, and used `nat.factors` in several later lemmas accordingly. I also removed `@[simp]`, but I wouldn't mind putting it back (or someone else putting it back).